lib/fetcher.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const co = require('co')
+
+function * addon (heroku, app, db) {
+  const {resolve} = require('heroku-cli-addons')
+  let attachment = yield resolve.attachment(heroku, app, db)
+  return yield heroku.get(`/addons/${attachment.addon.id}`)
+}
+
+function * all (heroku, app) {
+  const sortBy = require('lodash.sortby')
+  const notDatabaseUrl = (a) => !a.config_vars.find((c) => c === 'DATABASE_URL')
+
+  // TODO: make this more efficient without having to call /addons
+  // (will require an API change of some sort)
+  let {attachments, addons} = yield {
+    attachments: heroku.get(`/apps/${app}/addon-attachments`),
+    addons: heroku.get('/addons')
+  }
+  addons = addons.filter(a => a.addon_service.name === 'heroku-postgresql')
```

Comment:
Can we support staging add-ons here? This would filter them out. Right now support is kinda janky based on env vars (I think `HEROKU_POSTGRESQL_HOST` and `HEROKU_POSTGRESQL_ADDON_NAME` or something like that), but it works. I think something more robust would be harder, but porting the existing mechanism would be great.